addons/mail/static/src/models/message_action_view.js

```diff
@@ -175,6 +175,9 @@ registerModel({
                     case this.messageAction.messageActionListOwnerAsToggleCompact:
                         return this.env._t("Compact");
                     case this.messageAction.messageActionListOwnerAsToggleStar:
+                        if (this.messageAction.messageActionListOwner.message.isStarred) {
+                            return this.env._t("Remove from Todo");
```

Comment:
you need to add the new text to the pot file

Reply:
I exported a new pot file and added it to the diff. You may want to double check it as it also modifies activity stuff.